carma_wm/src/WMListenerWorker.cpp

```diff
@@ -429,6 +429,42 @@ double WMListenerWorker::getConfigSpeedLimit() const
   return config_speed_limit_;
 }
 
+std_msgs::Int32MultiArray WMListenerWorker::getIntersectionGroupIdsByCurLoc(const geometry_msgs::PoseStamped& current_pos)
+{  
+  //Get all traffic light regulatory elements along the route based on current SV position
+  lanelet::BasicPoint2d cur_loc;
+  cur_loc.x() = current_pos.pose.position.x;
+  cur_loc.y() = current_pos.pose.position.y;
+  std::vector<lanelet::CarmaTrafficLightPtr> light_v = world_model_->getLightsAlongRoute(cur_loc);
+  lanelet::CarmaTrafficLightPtr first_light = light_v.front();
```

Comment:
we should check if the returned list is empty or not before getting front() element, otherwise it might segfault

Reply:
Addressed https://github.com/usdot-fhwa-stol/carma-platform/pull/1425/commits/ca7ac52778215882aedfe5cabc8f7803fea80f98